config/config.txt

```diff
@@ -187,6 +187,12 @@ TICKLAG 0.5
 ## Name that the server calls itself in communications
 #CROSS_COMMS_NAME
 
+## Hub address for tracking stats
+MEDAL_HUB_ADDRESS
+
+## Password for the hub page
+MEDAL_HUB_PASSWORD
+
```

Comment:
I actually thought the same thing when I first was setting up the gloom hub page, but when you actually make it and look at the options, the hub address and password fields are clearly marked.
I can try adding an example for the address I guess. It would be ckey.hubname